doc/classes/SpringArm.xml

```diff
@@ -33,17 +41,26 @@
 			<argument index="0" name="RID" type="RID">
 			</argument>
 			<description>
+				Removes the given objects from the objects excluded from the collision check.
```

Comment:
"Removes the given object", also singular.

Reply:
Missed this one too